### DISCUSSION TITLE
How navigate to jdk source using gd like any other java source file within a project

### DISCUSSION THREAD

**Discussion by rohitvvv** [mfussenegger/nvim-jdtls#255] [Q&A]
I have a gradle project which builds fine. I am able to use gd to jump around. However when my cursor is on a jdk file it does not work. Any way to get it to work.

**Comment by mfussenegger** [comment]
Did you see the [troubleshooting section](https://github.com/mfussenegger/nvim-jdtls#troubleshooting) ?

You'll have to provide some more information - see also https://zignar.net/2021/12/03/help-people-help-you-and-put-in-some-effort/
